ScoutSuite/output/data/html/partials/aws/services.dynamodb.regions.id.tables.html

```diff
@@ -0,0 +1,29 @@
+
+<!-- Table partial -->
+<script id="services.dynamodb.regions.id.tables.partial" type="text/x-handlebars-template">
+  {{#unless scout2_link}}
+    <div id="resource-name" class="list-group-item active">
+      <h4 class="list-group-item-heading">{{TableName}}</h4>
+    </div>
+    <div class="list-group-item">
+    <h4>Information</h4>
+      <ul>
+        <li class="list-group-item-text">Id: {{TableId}}</li>
+        <li class="list-group-item-text">Arn: {{TableArn}}</li>
+        <li class="list-group-item-text">Automatic backups enabled: {{automatic_backups_enabled}}</li>
```

Comment:
You're missing the id_suffix.

```suggestion
        <li class="list-group-item-text">Automatic backups enabled: <span id="dynamodb.regions.{{region}}.tables.{{@key}}.automatic_backups_enabled">{{automatic_backups_enabled}}</span></li>
```